client/src/app/site/pages/meetings/modules/poll/components/single-option-chart-table/single-option-chart-table.component.html

```diff
@@ -18,13 +18,13 @@
                         </td>
                         <td>
                             <div class="result-cell-definition">
-                                <span *ngIf="row.value[0].showPercent">
+                                <span *ngIf="row.value[0].showPercent" [class]="'text-' + iconSize">
```

Comment:
You've used this exact string concatenation a whole five times in this PR, why not write a getter for it?